### PR TITLE
Add kmt.status task

### PR DESCRIPTION
**Pull request by gjulianm**
### What does this PR do?

This PR adds a `kmt.status` task that shows the status of all stacks and VMs.

### Motivation

Improving visibility into the KMT resources deployed, make it easier to know whether things are up or not.

### CODE REVIEW

File: tasks/kmt.py [hunk 1]
```diff
@@ -669,3 +679,90 @@ def ssh_config(
...
                 print("")
+
+
+@task(
+    help={
+        "stack": "Name of the stack to get the status of. If None, show status of stack associated with current branch",
+        "all": "Show status of all stacks. --stack parameter will be ignored",
+    }
+)
+def status(ctx: Context, stack: Optional[str] = None, all=False):
```
Comment (usamasaqib) [inline tasks/kmt.py:690]: Can we add a `verbose` option to also print exactly which VMs/Instances are up/down?

**Review by usamasaqib — APPROVED** [review]
Looks good!

File: tasks/kmt.py (hunk 1 above)
Comment (gjulianm) [reply tasks/kmt.py:690]: The current version should print the status of each VM/instance too without extra options, let me check

Comment (gjulianm) [reply tasks/kmt.py:690]: Screenshot 2024-03-22 at 14 25 53 

This is the output, I don't know if you were referring to something like this

Comment (usamasaqib) [reply tasks/kmt.py:690]: Yeah this is exactly what I meant

**Pull request merged**